tests/_data/config_default.php

```diff
@@ -0,0 +1,12 @@
+<?php
```

Comment:
This file is obsolete, if you remove the vfsStream.

Reply:
removed